guide-for-tag-members.md

```diff
@@ -52,6 +52,10 @@ We tend to look for issues like:
 
+#### Fast Track
+
+Optionally, when a design review comes in, a TAG member may mark it as `fast track?` and alert other TAG members. Fast track reviews are reviews that would be proposed to not take up call time. It’s up to the TAG member proposing fast track status to indicate what outcome they think the review should have. E.g. “This doesn’t need a review. It’s a small change to a spec we’ve already reviewed with no architectural issues. Suggested outcome: Resolution: Satisfied”. They should also add themselves to the issue and add a “Fast track?” label. If it gets at least 2 indications of support (e.g. thumbs up on Slack) (over 1 week from the time it has been proposed) from any other TAG members then it can be closed. It’s up to whoever proposed this for fast rack to close it and to write a brief closing note into the issue, summarizing the reason it can be closed, and label it appropriately. If other TAG members object over the 2 day period, then it goes into the regular review cycle
```

Comment:
```suggestion
Optionally, when a design review comes in, a TAG member may mark it as `fast track?` and alert other TAG members. Fast track reviews are reviews that would be proposed to not take up call time. It’s up to the TAG member proposing fast track status to indicate what outcome they think the review should have. E.g. “This doesn’t need a review. It’s a small change to a spec we’ve already reviewed with no architectural issues. Suggested outcome: Resolution: Satisfied”. They should also add themselves to the issue and add a “Fast track?” label. If it gets at least 2 indications of support (e.g. thumbs up on Slack) (over 1 week from the time it has been proposed) from any other TAG members then it can be closed. It’s up to whoever proposed this for fast rack to close it and to write a brief closing note into the issue, summarizing the reason it can be closed, and label it appropriately. If other TAG members object over the 1 week period, or it does not get sufficient support, then it goes into the regular review cycle.
```